packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -334,7 +334,16 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
         this.requestOpsNoncePrefix = `${this.documentId}-`;
     }
 
-    public requestOps(from: number, to: number) {
+    /**
+     * Retrieves ops from PUSH
+     * @param from - inclusive
+     * @param to - exclusive
+     * @returns ops retrieved
+     */
+     public requestOps(from: number, to: number) {
+        // Given that to is exclusive, we should be asking for at least something!
+        assert(to > from + 1, "empty request");
```

Comment:
Should this be `to > from`? If this gets called with something like `requestOps(1, 2)`, this assert would throw but it would be a valid request, right (should just give you the [1, 1] interval) ?

Reply:
You are right, I over did it :)